src/actions/index.js

```diff
@@ -135,16 +135,15 @@ const tryUpdate = async (client, instance, { retries = 0 }) => {
   try {
     return await cozyFetch(client, 'PUT', '/settings/instance', instance)
   } catch (error) {
-    const isConflictError =
-      error.errors && error.errors[0] && error.errors[0].status === '409'
+    const isConflictError = error.status === 409
```

Comment:
Does the fact that you know use `error.status` instead of `error.errors[0].status` means that this code did not work before?

Reply:
This code worked before I modified cozyFetch to use cozy-client under the hood. The code would look like ```
error?.reason?.errors?.[0]?.status
```
I chose to use directly status from the query to simply the code as it return the same result